urgent-care/pom.xml

```diff
@@ -94,7 +94,7 @@
           <execution>
             <phase>compile</phase>
             <configuration>
-              <tasks>
+              <target>
```

Comment:
I was trying to figure out why this changed and if this is true 
```This allows Maven to run Ant tasks. To do so, there must be an existing project and maven-antrun-plugin must have its <target> tag configured (although it would still execute without the <target> tag, it would not do anything).```

How were our openapi files not busted for the past couple months?

Reply:
Best I can tell is <task> is same as <target> but <task> is deprecated.  So I just changed it.
https://maven.apache.org/plugins/maven-antrun-plugin/run-mojo.html#tasks